business_requirement_deliverable_cost/models/business.py

```diff
@@ -122,7 +122,7 @@ class BusinessRequirementDeliverable(models.Model):
     )
 
     @api.multi
-    @api.depends('resource_ids')
+    @api.depends('resource_ids', 'resource_ids.price_total')
```

Comment:
@victormartinelicocorp This is documented in the conversation but not in a formal issue.
I would keep it here for the sake of simplicity